### PR TITLE
fix(ET-1265): Updates migration service documentation

### PR DESCRIPTION
**Pull request by ksnavely**
## What Changed?

I've updated some documentation content based on testing and review.

## Not In Scope

Anything unrelated to MP/MTK schema management. I made some small updates but am stopping to avoid scope creep.

### CODE REVIEW

File: product_docs/docs/transporter/ba_preview/limitations.mdx [hunk 1]
```diff
@@ -19,6 +19,6 @@ Unsupported Oracle data types include:
 - XML
 - Spatial
 
-Don't use EDB Transporter for replicating Oracle tables that contain blob, clob, or nclob columns and are expected to get UPDATE changes. Streaming UPDATE changes currently aren't handled correctly.
+Don't use EDB Transporter for replicating Oracle tables that contain BLOB, CLOB, or NCLOB columns that are expected to change. These types are supported for snapshots but are not captured in the streaming CDC replication.
```
Comment (ksnavely) [inline product_docs/docs/transporter/ba_preview/limitations.mdx:22]: @yidian1997 did I get this right?

Comment (yidian1997) [reply product_docs/docs/transporter/ba_preview/limitations.mdx:22]: Here is a chart I added to our team doc. We support streaming LOB for tables containing **primary key**. [Jira Issue](https://enterprisedb.atlassian.net/browse/ET-335)
For tables that don't have PK and have LOB type, we only support insert op.

**Comment by gvasquezvargas** [comment]
Hi all! There is some overlap with https://github.com/EnterpriseDB/docs/pull/5883/files. 
Specifically regarding the handling of the database schema. Let's have a sync and decide how to best walk the customer through the preparation of database schemas.

File: product_docs/docs/transporter/ba_preview/getting_started/prepare_schema.mdx [hunk 2]
```diff
@@ -2,4 +2,52 @@
 title: "Preparing schema"
 ---
 
-Use [EDB Migration Portal](https://www.enterprisedb.com/docs/migration_portal/latest/) to assess Oracle database sources for schema compatibility before starting the data migration process. Address incompatible schemas or exclude them from the migration and migrate the schema to the new database **without applying constraints**.
\ No newline at end of file
+# Oracle to EPAS Migrations
```
Comment (djw-m) [inline product_docs/docs/transporter/ba_preview/getting_started/prepare_schema.mdx:5]:
```suggestion
## Oracle to EPAS Migrations
```

No H1's in body text, start at H2 always.

File: product_docs/docs/transporter/ba_preview/getting_started/prepare_schema.mdx [hunk 3]
```diff
@@ -2,4 +2,52 @@
...
+
+Use [EDB Migration Portal](https://www.enterprisedb.com/docs/migration_portal/latest/) to assess Oracle database sources for schema compatibility before starting the data migration process. 
+
+EDB's Migration Portal offers the ability to [migrate database schema](https://www.enterprisedb.com/docs/migration_portal/latest/04_mp_migrating_database/03_mp_schema_migration/) through either an offline file export or a live application to the destination cluster.
+
+# Other Migrations
```
Comment (djw-m) [inline product_docs/docs/transporter/ba_preview/getting_started/prepare_schema.mdx:11]:
```suggestion
## Other Migrations
```

File: product_docs/docs/transporter/ba_preview/getting_started/prepare_schema.mdx [hunk 4]
```diff
@@ -2,4 +2,52 @@
...
+
+For migrations to and from Postgres we recommend using [EDBs Migration Toolkit](https://deploy-preview-5759--edb-docs-staging.netlify.app/docs/migration_toolkit/latest/) to manage schema. MTK's [offline migration](https://www.enterprisedb.com/docs/migration_toolkit/latest/07_invoking_mtk/08_mtk_command_options/#offline-migration-options) capability provides an easy way to extract database schema.
+
+Tools such as `pg_dump` and `pg_restore` are another valid route for migrating DDL.
+
+# Destination Database Schema Integrity and Performance Considerations
```
Comment (djw-m) [inline product_docs/docs/transporter/ba_preview/getting_started/prepare_schema.mdx:17]:
```suggestion
## Destination Database Schema Integrity and Performance Considerations
```

File: product_docs/docs/transporter/ba_preview/getting_started/prepare_schema.mdx [hunk 5]
```diff
@@ -2,4 +2,52 @@
...
+
+The presence of destination database constraints, triggers, and WAL logging can impact migration performance. If the target database has limited resources or resource contention is detected during migration consider delayed application of database constraints.
+
+The impact of various constraints and settings are described below.
+
+## Identity and Deduplication constraints
```
Comment (djw-m) [inline product_docs/docs/transporter/ba_preview/getting_started/prepare_schema.mdx:23]:
```suggestion
### Identity and Deduplication constraints
```

File: product_docs/docs/transporter/ba_preview/getting_started/prepare_schema.mdx [hunk 6]
```diff
@@ -2,4 +2,52 @@
...
+
+`PRIMARY KEY`
+`UNIQUE`
+
+These types of constraints are implicitly leveraged by EDB Transporter to provide an exactly-once-delivery guarantee when syncing data to the destination database. `PRIMARY KEY` and `UNIQUE` constraints should be applied before the data migration begins, while other types of constraints can be withheld until after data migration completes. `PRIMARY KEY` or `UNIQUE` is sufficient to achieve exactly-once-delivery.
+
+For rows in tables that do not have `PRIMARY KEY` or `UNIQUE` constraints it is only possible to achieve at-least-once delivery. Deduplication can be performed during integrity validation such as with [EDB LiveCompare](https://www.enterprisedb.com/docs/livecompare/latest/).
+
+## Performance Impacting Constraints
```
Comment (djw-m) [inline product_docs/docs/transporter/ba_preview/getting_started/prepare_schema.mdx:32]:
```suggestion
### Performance Impacting Constraints
```

File: product_docs/docs/transporter/ba_preview/getting_started/prepare_schema.mdx [hunk 7]
```diff
@@ -2,4 +2,52 @@
...
+
+In particular, data migration can only proceed in serial if `FOREIGN KEY` constraints are present in the destination tables.
+
+Other types of constraints lead to unnecessary CPU and memory utilization given the context of an in-flight data migration from a consistent and referentially integral source database.
+
+For Oracle sources we recommend Migration Portal be used to separate constraints from other destination DDL. This is supported in the [offline migration experience](https://www.enterprisedb.com/docs/migration_portal/latest/04_mp_migrating_database/03_mp_schema_migration/#offline-migration).
+
+For other sources Migration Toolkit can be used with `-offlineMigration` can be used to [extract source schema](https://www.enterprisedb.com/docs/migration_toolkit/latest/07_invoking_mtk/08_mtk_command_options/#offline-migration-options). MTK automatically separates constraints from other DDL objects in the exported DDL.
+
+## Less Impactful Constraints
```
Comment (djw-m) [inline product_docs/docs/transporter/ba_preview/getting_started/prepare_schema.mdx:51]:
```suggestion
### Less Impactful Constraints
```

File: product_docs/docs/transporter/ba_preview/index.mdx [hunk 8]
```diff
@@ -3,7 +3,7 @@ title: EDB Transporter
 indexCards: simple
 deepToC: true
 directoryDefaults:
-  description: "EDB Transporter is a cloud-first migration solution based on Debezium and Kafka. Change-event data streaming enables secure, fault-tolerant, and performant migrations to BigAnimal, on-premise EPAS/Postgres, or other cloud-hosted databases."
+  description: "EDB Transporter is our PG AI integrated migration solution based on Debezium and Kafka. Change-event data streaming enables secure, fault-tolerant, and performant migrations to BigAnimal, on-premise EPAS/Postgres, or other cloud-hosted databases."
```
Comment (djw-m) [inline product_docs/docs/transporter/ba_preview/index.mdx:6]: No use of first person as per style guide.

Also this is somewhat long for a description which is meant to be in an index card box.

Also, ~BigAnimal~ - EDB Postgres AI Cloud Service (or Cloud Service)
```suggestion
  description: "EDB Transporter is a PG AI integrated migration solution enabling secure, fault-tolerant, and performant migrations to EDB Postgres AI Cloud Service, on-premise EPAS/Postgres, or other cloud-hosted databases."
```

**Review by djw-m — COMMENTED** [review]
Some mismatches with style guide, use of BigAnimal as a product name, structural error in headings (can lead to bad SEO with multiple H1s), overlong description. Changes suggested.

**Comment by gvasquezvargas** [comment]
Closing in favor of https://github.com/EnterpriseDB/docs/pull/5922

**Pull request closed without merging**